cchecker.py

```diff
@@ -78,7 +92,7 @@ def main():
     parser.add_argument('-l', '--list-tests', action='store_true',
                         help='List the available tests')
 
-    parser.add_argument('-d', '--download-standard-names',
+    parser.add_argument('-d', '--download-standard-names', nargs='?',
```

Comment:
Hmm.  Don't remember adding this.  I must have accidentally added it when trying to modify